event_dispatcher/before_after_filters.rst

```diff
@@ -189,7 +189,7 @@ serve as a basic flag that this request underwent token authentication::
     {
         // ...
 
-        if ($controller[0] instanceof TokenAuthenticatedController) {
```

Comment:
This change makes me a bit nervous ... but I never know the exact format of controllers (I know it's not always an array, but sometimes it is). Any thoughts? Thanks!

Reply:
In the previous example (line 137 of this file), array controllers are already taken care of. So this is a correct change imho

Reply:
You are right! Sorry I missed this!